grafana-plugin/src/models/mattermost/mattermost_channel.ts

```diff
@@ -21,6 +26,28 @@ export class MattermostChannelStore extends BaseStore {
     this.path = '/mattermost/channels/';
   }
 
+  @action.bound
+  async updateMattermostChannels() {
+    const response = await makeRequest(this.path, {});
```

Comment:
It would be good to provide the type in here to so we have a typed response `await makeRequest<MattermostChannel[]>`